src/tiles/storage.rs

```diff
@@ -26,15 +26,17 @@ impl TileStorage {
     /// position.
     ///
     /// Panics if the given `tile_pos` doesn't lie within the extents of the underlying tile map.
-    pub fn get(&self, tile_pos: &TilePos) -> Option<Entity> {
+    pub fn get<T: Into<TilePos>>(&self, tile_pos: T) -> Option<Entity> {
```

Comment:
should this be part of a separate PR, where maybe this pattern is used more often?